src/lake_modelling/utils/test_model.py

```diff
@@ -25,29 +25,29 @@ def test_lake_monthly_flows(self):
         assert isinstance(test_lake.monthly_flows, dict)
         assert test_lake.monthly_flows[month] == flow
 
-    def test_valid_lake_pH_0_A_pH(self):
-        lake_1 = Lake(pH_lake0=4.5, colour_lake0=0.5)
-        lake_2 = Lake(pH_lake0=4.5, colour_lake0=3)
-        lake_3 = Lake(pH_lake0=4.5, colour_lake0=7)
-        lake_4 = Lake(pH_lake0=5.5, colour_lake0=0.5)
-        lake_5 = Lake(pH_lake0=5.5, colour_lake0=4)
-        lake_6 = Lake(pH_lake0=5.5, colour_lake0=21)
-
-        lake_7 = Lake(pH_lake0=2, colour_lake0=21)
-        lake_8 = Lake(pH_lake0=5.5, colour_lake0=15)
-
-        good_lakes = [lake_1, lake_2, lake_3, lake_4, lake_5, lake_6]
-        bad_lakes = [lake_7, lake_8]
-
-        for lake in good_lakes:
-            assert isinstance(lake.pH_0, float)
-            assert isinstance(lake.A_pH, float)
-
-        for lake in bad_lakes:
-            with pytest.raises(ValueError):
-                lake.pH_0
-            with pytest.raises(ValueError):
-                lake.A_pH
+#     def test_valid_lake_pH_0_A_pH(self):
```

Comment:
When I run pytest, 2 out of 11 tests fail:

```
def test_C_inst0(self):
>       assert round(test_model.C_inst0, 6) == 1.993075
E        assert 0.996537 == 1.993075
```

```
def test_C_bott0(self):
>       assert round(test_model.C_bott0, 6) == 0.623098
E       assert 0.311549 == 0.623098
```

Should we change the expected values for these tests?

Reply:
Oops, good catch, thanks!

Yes, the expected values should be updated - I've changed them in babd33ae5a09c208ea19b84fbdc836b23ba4f8d0. They've changed because the updated model introduces a new parameter named `spr_prop` (the proportion of the lake surface area that is limed) with a default value of 0.5. Because these tests assume a default model with a lime dose of 10 mg/l, the overdosing factor is 1, so the effect of `spr_prop` is just to make `C_inst0` and `C_bott0` half of what they were before.

Thanks for the review :-)